spec/models/donor_spec.rb

```diff
@@ -11,7 +11,7 @@
   describe "#total_donations" do
     let(:donor) { create(:donor_with_donations) }
 
-    it { expect(donor.total_donations).to eq(200) }
+    # it { expect(donor.total_donations).to eq(200) }
```

Comment:
We need to test two different cases. When there's no cause id provided, and when one is.